vpr/src/route/rr_graph.cpp

```diff
@@ -665,6 +666,7 @@ static void build_rr_graph(const t_graph_type graph_type,
                            const std::vector<t_physical_tile_type>& types,
                            const DeviceGrid& grid,
                            t_chan_width nodes_per_chan,
+                           t_router_opts router_opts,
```

Comment:
Another pass by value that should be a pass by reference, these are things you should always be on the lookout for!